app/views/api/user_preferences/index.json.jbuilder

```diff
@@ -0,0 +1 @@
+json.partial! @user_preferences
```

Comment:
This should also contain the `root_elements`, similar to the XML response. This is already implemented for e.g. showing relations - see https://github.com/openstreetmap/openstreetmap-website/blob/master/app/views/api/relations/show.json.jbuilder

When those are included, it becomes clearer that there needs to be a root `preferences` (n.b. plural) key, to match the naming of the `<preferences>` element in the XML response. 

So something like this:

```
{
    "version": "0.6",
    "generator": "OpenStreetMap server",
    "copyright": "OpenStreetMap and contributors",
    "attribution": "http://www.openstreetmap.org/copyright",
    "license": "http://opendatacommons.org/licenses/odbl/1-0/",
    "preferences": {
        "foo": "bar",
        "another_pref": "something else",
        "more preferences": "go here"
    }
}
```

Reply:
I should also say that the preferences are analogous to "tags" on nodes/ways/relations, in that they are already implemented in the nodes/way/relations responses so you can see how it's done there, and for json we already decided to drop the "k/v" stuff so as @pnorman mentioned it's fine to use object attributes directly for them here too.

Reply:
Thanks, I’ll add these! Do you know of a way to see multiple preferences? So far I’ve only been able to view the editor language, and I’d like to have additional key/value pairs.

Reply:
You'll have to upload preferences using [the api](https://wiki.openstreetmap.org/wiki/API_v0.6#Preferences_of_the_logged-in_user)